source/d_threedi_portal.rst

```diff
@@ -8,14 +8,14 @@ Login
 
 Follow the steps below to access the 3Di portal:
 
-#) Start the internet browser (Chrome or Firefox) and go to: https://3di.lizard.net/ 
+#) Start the internet browser (Chrome or Firefox) and go to: https://3di.live
 #) Log in with your username and password.
 
 	* Username and password can be requested via servicedesk@nelen-schuurmans.nl.
 	* You can clear the fields by clicking the **Reset values** button.
 	* Forgot your password? Request a new password via **Reset my password**.
 
-.. figure:: image/d2.2_login.png
+.. figure:: image/d2.1_login.png
```

Comment:
Consider using .. image:: tags when there is no caption needed. This also prevents possible rendering of normal text as caption for an image.

Reply:
I don't understand this, can you give me an example?

Reply:
See this bit of documentation telling about the differences:

https://docutils.sourceforge.io/docs/ref/rst/directives.html#image

Note how the .. figure:: tag has an option for a caption, while the .. image:: tag does not.